packages/snowpack/src/config.ts

```diff
@@ -230,24 +231,28 @@ function loadPlugins(
   function loadPluginFromConfig(name: string, options?: any): SnowpackPlugin {
     const pluginLoc = require.resolve(name, {paths: [process.cwd()]});
     const pluginRef = require(pluginLoc);
-    const plugin = pluginRef.default
-      ? pluginRef.default(config, options)
-      : pluginRef(config, options);
+    let plugin: SnowpackPlugin & LegacySnowpackPlugin;
+    try {
+      plugin = pluginRef.default ? pluginRef.default(config, options) : pluginRef(config, options);
+    } catch (err) {
+      console.error(`[${name}] ${err}`);
+      throw err;
```

Comment:
Now we surface plugin loading errors to the user a bit better (plugins can also validate their options)